core/domain/exp_domain.py

```diff
@@ -2101,6 +2110,71 @@ def _convert_states_v25_dict_to_v26_dict(cls, states_dict):
 
         return states_dict
 
+    @classmethod
+    def _convert_states_v26_dict_to_v27_dict(cls, states_dict):
+        """Converts from version 26 to 27. Version 27 adds content_translations
+        dict to the state, which will allow translators to add translation
+        script for the state contents.
+
+        NOTE: This migration will also filter out the content_id from
+        content_ids_to_audio_translations such that the state passes the new
```

Comment:
Give a bit more detail: what is the "new validation check"? I don't understand this paragraph.

EDIT: ah, the note extends to the next paragraph. I suggest combining the two.